README.md

```diff
@@ -39,3 +45,92 @@ Engine, enter "http://localhost:8181/ws".
 [system]: https://github.com/snaekobbi/system
 [ansible]: http://www.ansible.com
 [vagrant]: https://www.vagrantup.com/
+
+
+## Docker image
+
+
+### Building the image
+
+If you want to build the image yourself, you can run this command:
+
+```
+docker build .
+```
+
+The last line will display the id of the image that was built. Something like:
+
+```
+Successfully built 72ca1a0390e8
+```
+
+In this case, `72ca1a0390e8` is the image ID.
+
+@josteinaj has uploaded pre-built images to https://hub.docker.com/r/josteinaj/snaekobbi/ which
+means you can use those instead by simply referring to `josteinaj/snaekobbi` instead of
+building your own image. That image will be used in the following commands.
+
+**TODO**: set up automatic building of `snaekobbi/system` images instead.
```

Comment:
Well... as soon as it is merged into `master`; this PR merges into the `release/1.7.0` branch since `master` is a bit behind.